cypress/e2e/tests/setup/rancher-setup.spec.ts

```diff
@@ -45,4 +45,9 @@ describe('Rancher setup', () => {
     //   cy.url().should('equal', `${ Cypress.config().baseUrl }/home`);
     // });
   });
+
+  it('Create standard user after login', () => {
+    cy.login();
+    cy.createUser('standard_user', 'user');
```

Comment:
For lack of a better solution, can you add a comment here saying this should match the `TEST_USERNAME` env var used when running non-admin tests? That means anyone searching the code base for that env var name will get this place where it's used

Reply:
sure thing